src/api/app/controllers/webui/staging_workflows/ignored_requests_controller.rb

```diff
@@ -0,0 +1,57 @@
+module Webui
+  module StagingWorkflows
+    class IgnoredRequestsController < WebuiController
+      layout 'webui2/webui'
+
+      before_action :require_login
+      before_action :set_bootstrap_views
+      before_action :set_staging_workflow
+      before_action :set_ignored_request, only: [:destroy]
+      after_action :verify_authorized, except: [:index]
+
+      def index
+        @ignored_requests = @staging_workflow.ignored_requests
+      end
+
+      def create
+        authorize @staging_workflow
+
+        ignored_request = IgnoredRequest.new(bs_request_id: params[:ignored_request][:bs_request_id], description: params[:ignored_request][:description])
+        ignored_request.staging_workflow = @staging_workflow
```

Comment:
Then you can just save it right away: `@staging_workflow.ignored_requests.create(attributes...) `

Reply:
@ChrisBr:
> I think we should create it like this:
> 
> ```
> @staging_workflow.ignored_requests.build(attributes...)
> @staging_workflow.save
> ```

@bgeuken:
> Then you can just save it right away: `@staging_workflow.ignored_requests.create(attributes...) `

The option @ChrisBr proposed was chosen. `save` returns `true` or `false`, what is used to show an error or a success message. `create` returns the object.

Reply:
After discussing with @DavidKang this is what we chose:
```
request_ignoration = @staging_workflow.request_ignorations.build(request_ignoration_params)

f request_ignoration.save
```
This allows us:
1. to know if the status of the save resulted in an error or with success, and
2. if there is an error when saving, to know the exact reason. Otherwise it will only show a message of "Ignored request not valid".